src/components/Button/Button.jsx

```diff
@@ -17,7 +18,34 @@ export function Button({
   onClick,
   className,
   type,
+  to = false,
 } = {}) {
+  if (to)
+    return (
+      <Link
```

Comment:
With such structure it's ofc working, but I started wondering if it's semantically valid to nest `button` inside `a`, cause if I'm being honest... I don't know the answer :) 

I found some explanation in https://stackoverflow.com/questions/64443645/html-semantics-button-acting-as-an-anchor and the #3 option is appealing to me slightly better, it's to style `Link` to look like a button in that case. 

Wdyt about that? Maybe you know something more about which option should be semantically correct? If so - I'd be really happy if you'd explain it to me or point to some valuable resource explaining such a case.

Reply:
You are right